models/text_recognition_crnn/README.md

```diff
@@ -8,12 +8,10 @@ Results of accuracy evaluation with [tools/eval](../../tools/eval) at different
 | ------------ | ---------- | --------- | --------- |
 | CRNN_EN      | 81.66      | 74.33     | 52.78     |
 | CRNN_EN_FP16 | 82.01      | 74.93     | 52.34     |
-| CRNN_EN_INT8 | 81.75      | 75.33     | 52.43     |
 | CRNN_CH      | 71.28      | 80.90     | 67.36     |
 | CRNN_CH_FP16 | 78.63      | 80.93     | 67.01     |
-| CRNN_CH_INT8 | 78.03      | 80.93     | 66.67     |
 
-\*: 'FP16' or 'INT8' stands for 'model quantized into FP16' or 'model quantized into int8'
```

Comment:
Any context regarding the removal of int8 models?

Reply:
We decided to retain int8 models after disscussion, and I will re-upload the int8 models.

Reply:
I propose to keep the original int8 model, since the accuracy is the same.

Reply:
> retain int8 models

Okay

Reply:
Does the accuracy of EN_FP16 change since the model is updated?

Reply:
The accuracy is the same.